src/TourPortal.js

```diff
@@ -478,7 +478,7 @@ class TourPortal extends Component {
             {steps[current] &&
               (typeof steps[current].content === 'function'
                 ? steps[current].content({
-                    close: onRequestClose
+                    close: onRequestClose,
```

Comment:
Good catch! Would be nice to add a linter to this project to catch errors like this.

Reply:
absolutely agree! mind make a PR?